src/DynamoApplications/StartupUtils.cs

```diff
@@ -93,7 +96,8 @@ public static CommandLineArguments Parse(string[] args)
                 .Add("v=|V=", "Verbose, Instruct Dynamo to output all evalautions it performs to an xml file at this path", v => verbose = v)
                 .Add("x|X", "When used in combination with the 'O' flag, opens a .dyn file from the specified path and converts it to .json." + 
                 "File will have the .json extension and be located in the same directory as the original file.", x => convertFile = x != null)
-                .Add("h|H|help", "Get some help", h => showHelp = h != null);
+                .Add("h|H|help", "Get some help", h => showHelp = h != null)
+                .Add("g=|G=|geometry", "Geometry, Instruct Dynamo to output geometry from all evaluations to a json file at this path", g => geometryFilePath = g);
```

Comment:
could you update the dynamo CLI wiki page with an example of this format?

Reply:
Sure, I will make sure the wiki gets updated.